src/providers/tfc/runProvider.ts

```diff
@@ -44,6 +47,15 @@ export class RunTreeDataProvider implements vscode.TreeDataProvider<TFCRunTreeIt
         this.reporter.sendTelemetryEvent('tfc-runs-viewInBrowser');
         vscode.env.openExternal(run.websiteUri);
       }),
+      vscode.commands.registerCommand('terraform.cloud.run.viewPlan', async (plan: PlanTreeItem) => {
+        await vscode.commands.executeCommand('setContext', 'terraform.cloud.run.viewingPlan', true);
```

Comment:
Shouldn't this be after the check if no plan found?

Reply:
Yeah, can be - I didn't realise that the condition produces popup error message which is entirely independent of the tree view. Will fix that.